examples/l402.ts

```diff
@@ -0,0 +1,51 @@
+import { AlbyWallet, MemoryTokenStore } from "l402";
+const { Client } = require('@getalby/sdk');
+import { ClientOptions, Satscanner } from "../src";
+require('dotenv').config();
+
+// Load the Alby token from an environment variable
```

Comment:
The example is good.
Would be great to have a section in the main readme for L402 and link to this example.

Reply:
This could also have links or a summary explaining L402 in general

Reply:
That is a good idea.

Reply:
Added that to the README.